resources/application.conf

```diff
@@ -11,5 +11,6 @@ ktor {
 wallet {
     alreadySynced = false
```

Comment:
We don't need the `alreadySynced` configuration variable anymore. You can remove it from the config file and the `Application.kt` code.